oeps/architectural-decisions/oep-0014-proc-archive-repos.rst

```diff
@@ -172,46 +172,52 @@ Reasons rejected:
 Change History
 **************
 
-2017-01-18
+2022-09-07
```

Comment:
I appreciate inverting the change history but could that be in a separate commit?